site/installing.md

```diff
@@ -11,6 +11,7 @@ Weave Cloud is the recommended option if:
  * You are deploying to larger clusters.
  * You require secure remote access.
  * You want to share access with your coworkers.
+ * You want to minimize Weave Scope memory and CPU usage.
```

Comment:
_You want to minimize your local consumption CPU and memory consumption._ would be clearer